src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,129 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+
+    private Storage storage;
+    private TaskList tList;
+    private Ui ui;
+    private Boolean active;
+
+    /**
+     * Constructor for a Duke object.
+     *
+     * @param dirPath path of directory that storage use to locate save files.
```

Comment:
For Javadoc, it is good practice to begin your sentence with a capital letter, in this case should be "Path of directory..." 😃

Reply:
Fixed.